.github/workflows/build-with-bal-test-graalvm-template.yml

```diff
@@ -75,6 +67,21 @@ jobs:
               run: |
                   perl -pi -e "s/^\s*version=.*/version=${{ env.BALLERINA_LANG_VERSION }}/" gradle.properties
                   ./gradlew build -x check -x test publishToMavenLocal --scan --no-daemon
+                  
+            - name: Set up GraalVM
+              uses: graalvm/setup-graalvm@v1
+              with:
+                  java-version: '17'
```

Comment:
```suggestion
                  java-version: '17.0.7'
```

Reply:
Lets try with the latest version, if there is an issue will change this version